README.md

```diff
@@ -1,3 +1,9 @@
+![Build Status](https://img.shields.io/github/workflow/status/awslabs/karpenter/Presubmit)
+![GitHub stars](https://img.shields.io/github/stars/awslabs/karpenter)
+![GitHub forks](https://img.shields.io/github/forks/awslabs/karpenter)
+![GitHub License](https://img.shields.io/github/license/awslabs/karpenter)
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/awslabs/karpenter/issues)
+
```

Comment:
goreportcard would be nice too:

`https://goreportcard.com/badge/github.com/awslabs/karpenter`

Reply:
would also be nice to have code coverage, although that might require more changes to the build system?

Reply:
nice will add this one.

Reply:
> would also be nice to have code coverage, although that might require more changes to the build system?

yeah, will require to integrate with codecov or something similar, will need to spend sometime comparing whats the right way to do it.